cmsdb/campaigns/run2_2018_nano_v9/__init__.py

```diff
@@ -292,3 +295,93 @@
         ),
     ),
 )
+
```

Comment:
Could you move the following datasets to a file `top.py`?
Also, please prepend `mt` to the top mass values in dataset names.